src/components/Resources/SocialMediaLink.tsx

```diff
@@ -0,0 +1,10 @@
+interface ISocialMediaLink {
+    platform: string
+    value: string
+}
+
+export default function SocialMediaLink({ platform, value }: ISocialMediaLink) {
+    return <div className="d-flex flex-row gap-1">
```

Comment:
Formatting/Non-Blocking: This jsx should be wrapped in parens, and the `<i />` and `<a />` elements on separate lines. Do you have a prettierjs plugin installed locally? We use a prettier config that can format as you type to the linter/style spec.

Reply:
I think there should be a `target="_blank"` on the `<a>` element so the links open in a new tab.